src/core/public/notifications/toasts/error_toast.tsx

```diff
@@ -29,7 +30,7 @@ import {
   EuiModalHeaderTitle,
 } from '@elastic/eui';
 import { EuiSpacer } from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
```

Comment:
At some point, this stateful import will probably go away. Can we pass this Provider down from the I18n service to future-proof this a bit?

Reply:
Sure. However, the stateful import is also used in `kibana_react`'s`toMountPoint`
https://github.com/elastic/kibana/blob/e04adbe34b6364e67b2219d379fc2ec22f06d843/src/plugins/kibana_react/public/util/react_mount.tsx#L22-L40

Should this be a concern?